### ISSUE TITLE
Confirmation screen for attendance sign in/out

### ISSUE THREAD

**Issue by superelitist** [schmave/demschooltools#124]
Based on a second-hand description of the new attendance sign in/out process and how a user might accidentally choose the wrong name, it sounds like some sort of confirmation screen might be useful. Perhaps expanding the user's name to fullscreen with a yes/no dialogue, or even expanding a photo of the user?

**Comment by schmave** [comment]
Yes, I think that something like this may be helpful. Maybe even instead of clicking on a name to sign in, you enter a 3-digit code, and then click to confirm. Or in order to keep it fast to sign in, maybe enter a 3-digit code and only have to click if you realize that you made a mistake.

**Comment by superelitist** [comment]
I wonder how well a wide range of ages will remember a PIN?

**Comment by schmave** [comment]
I do too! :)

I will probably think more about this over the summer. If you make any mockups or do some user research or just have any other ideas, I'm curious to hear it/them.

**Comment by schmave** [comment]
Moved to Linear
